frontend/src/scenes/max/maxLogic.tsx

```diff
@@ -70,29 +66,42 @@ export const maxLogic = kea<maxLogicType>([
             ['dataProcessingAccepted', 'toolMap', 'tools'],
             maxSettingsLogic,
             ['coreMemory'],
+            sidePanelStateLogic,
+            ['sidePanelOpen'],
             // Actions are lazy-loaded. In order to display their names in the UI, we're loading them here.
             actionsModel({ params: 'include_count=1' }),
             ['actions'],
+            router,
+            ['location as routerLocation'],
```

Comment:
`router` isn't useful to connect, since it's always mounted, it's actually less clear this way I would say. Best to just always directly use `router.selectors.location` or `router.values.location`.